libgraphics/src/harfbuzztext.cpp

```diff
@@ -214,7 +214,7 @@ MCHarfbuzzSkiaFace *MCHarfbuzzGetFaceForSkiaTypeface(SkTypeface *p_typeface, uin
         t_hb_sk_face -> skia_face = t_face;
         p_typeface -> ref();
         
-		MCGCacheTableSet(s_hb_face_cache, t_key, sizeof(t_id), t_hb_sk_face, sizeof(MCHarfbuzzSkiaFace));
+        //MCGCacheTableSet(s_hb_face_cache, t_key, sizeof(t_id), &t_hb_sk_face, sizeof(MCHarfbuzzSkiaFace*));
```

Comment:
The problem is that sizeof(MCHarfbuzzSkiaFace) > sizeof(uintptr_t) so the MCGCacheTable code doesn't work at all - the implementation refuses to cache any objects over that size.